bundler.d/fog.rb

```diff
@@ -1,5 +1,5 @@
 group :fog do
- gem 'fog', '1.25.0'
- gem 'fog-core', '1.25.0'
+ gem 'fog', '1.27.0'
+ gem 'fog-core', '1.27.3'
  gem 'unf', '>= 0.1.3'
```

Comment:
unf has been dropped since https://github.com/fog/fog/commit/8426fc9abf10c0ce2b039d26830e000361dac604 it seems.